integ-test/src/test/java/org/opensearch/sql/legacy/OpenSearchSQLRestTestCase.java

```diff
@@ -66,6 +66,9 @@ protected RestClient buildClient(Settings settings, HttpHost[] hosts) throws IOE
   }
 
   protected static void wipeAllOpenSearchIndices() throws IOException {
+    if (client() == null) {
```

Comment:
How did you run into this failure?

Reply:
If test cancelled in `assume` `client()` returns `null`, because it not initialized yet.
https://github.com/Bit-Quill/opensearch-project-sql/blob/f0a3bdd817753b46517da07c1059c237501442eb/integ-test/src/test/java/org/opensearch/sql/ppl/ShowCatalogsCommandIT.java#L26